theme/layouts/_breadcrumbs.ejs

```diff
@@ -0,0 +1,21 @@
+<%
+const query = site.queryPages;
+if (page.pathname !== '') {
+  const [currentPage] = query(({pathname}) => page.pathname === pathname);
+  const pathnameParts = currentPage.originalPathname.split('/'); %>
+  
+  <nav aria-label='Breadcrumb' id='breadcrumb'>
+  <ol>
+    <% 
+    for (let i=0; i < pathnameParts.length; i++) {
+      const pickOriginalPathname = pathnameParts.slice(0, i).join("/");
+      const [{pathname, title}] = query(({originalPathname}) => 
+        originalPathname == pickOriginalPathname); %>
+        <li class='parts'>
+        <a href = '<%= pathname %>'><%= title %></a>
```

Comment:
Detail: HTML attributes doesn't suppose to have trailing spaces. `<a href='<%= pathname %>'><%= title %></a>`